units/ships.json

```diff
@@ -560,8 +560,7 @@
                         "Prohibited_Upgrades": "mult_shady_moreupgrade;mult_shady_morethrust;mult_shady_moreshields;mult_shady_moregunrecharge;mult_shady_moreshieldrecharge;mult_shady_moreturning;upgrades/Overdrive;upgrades/Capacitors:1;upgrades/Reactors:1;upgrades/Armor:1;upgrades/Sensors:1;upgrades/Shield_Systems:1;upgrades/Passenger_Quarters:1;upgrades/Repair_Systems:1;upgrades/ECM_Systems:1",
                         "Mounts": "{KtekBeam;;;Light Medium Heavy autotracking;-9;-.25;-.75;;;;;;;;;1;1}{KtekBeam;;;Light Medium Heavy autotracking;9;-.25;-.75;;;;;;;;;1;1}{Ktek;;;Light;-4.5;-.6;12.25;;;;;;;;;1;1}{Ktek;;;Light;4.5;-.6;12.25;;;;;;;;;1;1}{MiniGravThumper;8;200;special-missile;0;-0.5;0;;;;;;;;;1;1}",
                         "armor": "1400.0",
-                        "shield": "412.5",
-                        "shield_facets": "4"
```

Comment:
Just to make sure I understand correctly: We're moving `"shield_facets"` from the `"units"` section to the `"data"` section? And the same with the other ships?

Reply:
The file is hierarchical.
Attributes are always in the data section.
Ships inheriting from this ship are in the units array with their own data section (where different).

Because shield_facets was only generated for ships with shield, it only appeared for "*" or "*.milspec" for example.
Once we fixed this, the value is in the parent "*.blank" and therefore removed from the inheriting units.

Reply:
I see. Sounds good.